internal/agent/device/config/bootstrap.go

```diff
@@ -96,6 +97,24 @@ func (c *Controller) writeDeviceEnrollmentBanner(_ context.Context, device *v1al
 	if err := c.writeQRBanner("\nEnroll your device to flightctl by scanning\nthe above QR code or following this URL:\n%s\n\n", url); err != nil {
 		return fmt.Errorf("failed to write device enrollment banner: %w", err)
 	}
+
```

Comment:
Please move it to the writeQRBanner instead, otherwise once enrolled we will never notify on next restart, but otherwise we will do it when we write the management banner.

Reply:
https://github.com/flightctl/flightctl/pull/149/files#diff-7012d62d090ebd6a70117c0f57275b301bc6c86605ae42c51497463265d9b39aR134

Reply:
In the future the bootstrap logic will be moved before the controllers start so we will not wait for sync. But just to align on flow it will look like this. So regardless of enrolled or not once bootstrap is complete we notify.

<bootstrap>
<-- bootstrap complete -->
<notify>
<run agent>

Does that sound logical?

Reply:
makes sense, we reduce the initialization time. Also we may want to re-trigger bootstrap in some cases (may be simply by exiting the service and letting systemd restart)

Reply:
Agree, waiting 60 seconds for the machine to give a tty makes no sense... We should add that refactor to JIRA